test/unit/cli_tests.rb

```diff
@@ -29,22 +29,23 @@ class UnitTests < Assert::Context
     end
 
     should "know its commands" do
-      assert_equal 12, COMMANDS.size
+      assert_equal 6, COMMANDS.size
 
       assert_instance_of InvalidCommand, COMMANDS[Factory.string]
 
-      assert_equal GenerateCommand, COMMANDS['generate']
-      assert_equal GenerateCommand, COMMANDS['g']
-      assert_equal BuildCommand,    COMMANDS['build']
-      assert_equal BuildCommand,    COMMANDS['b']
-      assert_equal InstallCommand,  COMMANDS['install']
-      assert_equal InstallCommand,  COMMANDS['i']
-      assert_equal PushCommand,     COMMANDS['push']
-      assert_equal PushCommand,     COMMANDS['p']
-      assert_equal TagCommand,      COMMANDS['tag']
-      assert_equal TagCommand,      COMMANDS['t']
-      assert_equal ReleaseCommand,  COMMANDS['release']
-      assert_equal ReleaseCommand,  COMMANDS['r']
+      assert_instance_of GenerateCommand, COMMANDS['generate']
+      assert_instance_of BuildCommand,    COMMANDS['build']
+      assert_instance_of InstallCommand,  COMMANDS['install']
+      assert_instance_of PushCommand,     COMMANDS['push']
+      assert_instance_of TagCommand,      COMMANDS['tag']
+      assert_instance_of ReleaseCommand,  COMMANDS['release']
+
+      assert_same COMMANDS['generate'], COMMANDS['g']
+      assert_same COMMANDS['build'],    COMMANDS['b']
+      assert_same COMMANDS['install'],  COMMANDS['i']
+      assert_same COMMANDS['push'],     COMMANDS['p']
+      assert_same COMMANDS['tag'],      COMMANDS['t']
+      assert_same COMMANDS['release'],  COMMANDS['r']
```

Comment:
@jcredding I reworked these tests to make them more robust in testing the aliases btw.